checkov/common/checks_infra/solvers/resource_solvers/not_exists_resource_solver.py

```diff
@@ -9,5 +9,6 @@
 class NotExistsResourcerSolver(ExistsResourcerSolver):
     operator = Operators.NOT_EXISTS  # noqa: CCE003  # a static attribute
 
-    def get_operation(self, *args: Any, **kwargs: Any) -> bool:
-        return not super().get_operation(*args, **kwargs)
+    def get_operation(self, *args: Any, **kwargs: Any) -> Optional[bool]:
```

Comment:
why change this?

Reply:
because I wanted to have an option of `None` result which indicates that the resource "passed", and in this flow it means it should be skipped.
pls pay attention that `base_resource_solver` `run` and `_process_node` functions handle both `exists_resource_solver` and `not_exists_resource_solver`.
In `ExistsResourcerSolver.get_operation ` we can get either True/False, while in `NotExistsResourcerSolver.get_operation` we can get False/None